index.html

```diff
@@ -12,22 +12,31 @@
         <link rel="apple-touch-icon" href="apple-touch-icon.png">
 
         <link rel="stylesheet" href="css/bootstrap.min.css">
-        <style>
-            body {
+      <!--   <style>
+           body {
                 padding-top: 50px;
                 padding-bottom: 20px;
-            }
-        </style>
+            }  
+        </style> 
+        removed to fix top alignment of page -->
+        
         <link rel="stylesheet" href="css/bootstrap-theme.min.css">
         <link rel="stylesheet" href="css/main.css">
 
         <script src="js/vendor/modernizr-2.8.3-respond-1.4.2.min.js"></script>
     </head>
     <body>
-        <!--[if lt IE 8]>
+
+        <!--[if lt IE 8]>   <img src="BKG.png" alt="Background image">
             <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
-    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+        
+     <!-- suggest moving image below nav bar. -->
```

Comment:
I agree. The navbar should remain at the top of the page. I have no preference, fixed or floating.